### PR TITLE
feat(ANXS/postgresql): remove obsolete v9.6 and prior code

### PR DESCRIPTION
**Pull request by gclough**
BREAKING CHANGE: v9.6 is no longer supported

### CODE REVIEW

**Comment by gclough** [comment]
@maglub or @otakup0pe , could someone review this for me?  I'll probably remove v10 soon, as that's been desupported by PGDG.

**Review by UnderGreen — APPROVED** [review]

**Comment by UnderGreen** [comment]
I assume my approval has no power anymore, but yeah. Approved :-D

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
           - distro: centos8
           - distro: debian9
           - distro: debian10
-          - distro: fedora33
+          # - distro: fedora33  # Removing to test pipeline
```
[the remark this answers is not in the document]
Comment (gclough) [reply .github/workflows/ci.yml:50]: I don't think so... but I've no idea how to make it work.  The errors don't make sense.

As far as I know there's nothing in this MR that caused it to break, it's just that the pipeline is no longer working on that OS.  I wanted to get us back to a "green" status, then we can make a new MR to re-add Fedora.

**Review by otakup0pe — APPROVED** [review]

**Comment by MrMegaNova** [comment]
Thanks for you work, can you open a new PR with commit "[feat(ANXS/postgresql): remove obsolete v9.6 and prior code](https://github.com/ANXS/postgresql/pull/528/commits/1318b071eddd47d187f086e8fed00c4f168ac09f)" on development branch and fix conflicts ?

**Pull request closed without merging**